build-all.sh

```diff
@@ -1,15 +1,30 @@
 #!/bin/bash
 
-# Build two distribution files
+# Build two distribution files:
 #   artifacts/flutter-intellij.jar
 #   artifacts/flutter-studio.zip
 
+# fast fail the script on failures.
+set -e
+
+# clean the build
 rm -rf build/*
-ant -Ddart.plugin.version=171.4424.10 -Didea.version=171.4408382 -Didea.product=android-studio-ide \
-  -DDEPENDS=\<depends\>com.android.tools.apk\</depends\> \
-  -DPLUGINID=io.flutter.as -DSINCE=171.1 -DUNTIL=171.\*
+
+# build for Android Studio
+# TODO(devoncarew): The version range below (supported for Android Studio) is
+# a temporary measure.
+ant \
+  -Ddart.plugin.version=171.4424.10 \
+  -Didea.version=171.4408382 \
+  -Didea.product=android-studio-ide \
+  -DSINCE=171.1 -DUNTIL="173.*"
```

Comment:
Are you sure loading a plugin compiled for 2017.1 into 2017.2 is able to create projects?

Reply:
Yup! I did check in Android Studio, IntelliJ, and IntelliJ EAP. I was running through lots of builds locally, so definitely would be good to double-check after this lands (as part of the normal release process).

Reply:
Good to know, thanks!